### ISSUE TITLE
Bugs in adding and removing users

### ISSUE THREAD

**Issue by xet7** [wekan/wekan#4730]
UCS sent info about these bugs in WeKan:

Case 1)

User B invites User A to a board 1. User A accepts and is listed as
member on board 1. User A now sees board 1 in the personal overview.
User A gets removed from board 1 but still sees the board in the
personal overview.

Adding and removing user A again doesn't fix it. User A can't remove
the board from the personal overview.

Case 2)

User B invites User A to board 1. User A declines the invitation but
still sees board 1 in the personal overview.

I fixed the problem working directly on the database:

> db.boards.find({title: "<board>"}, {_id: 1})
> db.users.find({username:"<user>"}, {_id: 1})
> db.boards.update({ _id: "<board_id>"}, { $pull: {members: {userId:
> "<user_id>"}}});

We currently use this Wekan version:

> UCS: 4.4-9 errata1324
> Installed: prometheus-node-exporter=1.2 wekan=6.09
> Upgradable: 

Please provide an update where removing users from boards works using
the web interface :)

**Comment by rainbowjaw** [comment]
yeah i'm finding the interface for adding users to new boards is broken.

**Comment by DVNBLMHC** [comment]
Hi, the problem is still present. 

When I try to import an organization/team, the users didn´t get access to the board.
